internal/pkg/utils/deepcopy/deepcopy_test.go

```diff
@@ -13,13 +13,16 @@ import (
 	"github.com/keboola/keboola-as-code/internal/pkg/utils/testassert"
 )
 
+type Values []*Bar
+
 type Foo struct {
-	Values []*Bar
+	Values Values
```

Comment:
Pridal som do testu rovno aj vlastny slice type na otestovanie, ... ale to ide v pohode.